build.gradle

```diff
@@ -98,14 +99,26 @@ task downloadLiferayDefinitions(type: Download){
 	overwrite false
 }
 
+task downloadLiferayTaglibDefinitions(type: Download){
+	def target = new File("src/main/resources/tld");
+	target.mkdirs();
+
+	src([
+			'https://raw.githubusercontent.com/liferay/liferay-portal/master/util-taglib/src/META-INF/liferay-aui.tld',
+			'https://raw.githubusercontent.com/liferay/liferay-portal/master/util-taglib/src/META-INF/liferay-ui.tld'
+		])
+	dest target
+	overwrite false
+}
+
 task verifyBlade(type: Verify, dependsOn: downloadBlade) {
 	src 'src/main/resources/libs/blade.jar'
 	algorithm 'MD5'
 	checksum bladeLatestMD5
 }
 
 compileJava {
-	dependsOn verifyBlade, downloadLiferayDefinitions
+	dependsOn verifyBlade, downloadLiferayDefinitions, downloadLiferayTaglibDefinitions
```

Comment:
Instead of making compileJava depend on this task, you should do it in

`processResources.dependsOn downloadLiferayTaglibDefinitions`